network/kademlia_test.go

```diff
@@ -370,6 +372,51 @@ func binStr(a *BzzAddr) string {
 	return pot.ToBin(a.Address())[:8]
 }
 
+func TestEachConnLB(t *testing.T) {
```

Comment:
Comment test describing what it does, please. In particular that this only tests neighborhood.

Reply:
Yes, and I'm also missing capability tests and tests where in each "iteration" the caller uses more than one peer (for example in pss forwarding when sending a message to all peers in a bin)

Reply:
Yes, sir. Along with bin test unfiltered by capability.